src/dndbeyond/content-scripts/character.js

```diff
@@ -1212,7 +1212,9 @@ function rollSpell(force_display = false, force_to_hit_only = false, force_damag
         if (character.hasClassFeature("Hexblade’s Curse") &&
             character.getSetting("warlock-hexblade-curse", false))
             critical_limit = 19;
-        const roll_properties = buildAttackRoll(character,
+        if (spell_full_name === "Blade of Disaster")
+            critical_limit = 18;
+            const roll_properties = buildAttackRoll(character,
```

Comment:
messed the indentation here, which makes it look like the `buildAttackRoll` is called inside the if.

Reply:
woops